analysis/druidrestoration/src/modules/core/Rejuvenation.ts

```diff
@@ -33,36 +33,43 @@ class Rejuvenation extends Analyzer {
     healingDone: HealingDone,
     mastery: Mastery,
   };
+
+  protected healingDone!: HealingDone;
+  protected mastery!: Mastery;
+
   totalRejuvsCast = 0;
 
-  constructor(options) {
+  constructor(options: Options) {
```

Comment:
Commented out unused bits (below) to avoid "unused" warning. Will be revived in future work.